src/main/features/claim/form/models/claimAmountBreakdown.ts

```diff
@@ -68,14 +73,10 @@ export default class ClaimAmountBreakdown implements Serializable<ClaimAmountBre
       return ClaimAmountBreakdown.initialRows()
     }
 
-    let claimAmountRows: ClaimAmountRow[] = []
-
-    for (let row in rows) {
-      claimAmountRows.push(new ClaimAmountRow().deserialize(rows[row]))
-    }
+    let claimAmountRows: ClaimAmountRow[] = rows.map(row => new ClaimAmountRow().deserialize(row))
```

Comment:
I think this can be a const

Reply:
in line 78 we assign value to this variable